packages/strapi-plugin-content-manager/admin/src/containers/ListView/Filter.js

```diff
@@ -21,13 +21,20 @@ function Filter({
   let displayedValue = toString(value);
 
   if (type.includes('date')) {
-    const date = moment(value.slice(0, -1), moment.ISO_8601);
-    const format =
-      date.valueOf() === date.startOf('day').valueOf()
-        ? 'MMMM Do YYYY'
-        : 'MMMM Do YYYY, h:mm:ss a';
+    const date = moment(value, moment.ISO_8601);
 
-    displayedValue = date.format(format);
+    let format;
+
+    if (type === 'date') {
+      format = 'MM/DD/YY';
+    } else {
+      format = 'MM/DD/YY HH:MM';
+    }
```

Comment:
This format is also used for other languages?

Reply:
Yes it is but I think that I should used the same file where the date format are defined. I'll update the PR accordingly.